agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryChannelTest.java

```diff
@@ -76,7 +75,9 @@
   private TelemetryChannel getTelemetryChannel(boolean followInstrumentationKeyForRedirect)
       throws MalformedURLException {
     List<HttpPipelinePolicy> policies = new ArrayList<>();
-    policies.add(new RedirectPolicy(followInstrumentationKeyForRedirect));
+    StatsbeatModule mockStatsbeatModule = Mockito.mock(StatsbeatModule.class);
+    Mockito.doNothing().when(mockStatsbeatModule).sendNetworkStatsbeatOnRedirect(null);
```

Comment:
I think this is the default
```suggestion
```

Reply:
no, that won't work.. i tried.. that will require more mock object.. it will throw nullpointerexception when executing that method. if condition will blow up.  need to mock the whole chain of configuration.

Reply:
can you double check? that doesn't sound right, this is a mock, not a spy

Reply:
that was my first option. then switched to donothing(). it's a mock, not a spy, but this method works.